client/src/app/main/main.html

```diff
@@ -1,6 +1,7 @@
 <section class="jumbotron">
-  <h1>Receipt Yourself!</h1>
   <p class="lead">
-    <img src="assets/images/receipt-logo.png" alt="Receipt Yourself!" width=100>
+    <img src="assets/images/logo-receipt-yourself-transparent.png" alt="Receipt Yourself!">
   </p>
+  <hr/>
 </section>
+<sign-in ng-show="!user.id"></sign-in>
```

Comment:
Here is the explanation why this is not working http://stackoverflow.com/questions/23595866/why-cant-the-rootscope-be-accessed-in-the-template-of-a-directive-with-isolate

You should consider creating new `ui-view` just for rendering login form, beacuse it's scope specific (it puts `user` object inside `$rootScope`. Here is the example on how to put new `ui-view` https://github.com/nikolalsvk/receipt-yourself/blob/master/client/src/index.html#L33.

After that you should define what is rendered in that `ui-view` like here =>
https://github.com/nikolalsvk/receipt-yourself/blob/master/client/src/app/index.route.js